releases/sui/2025-01-ITS-v1.1.3.md

```diff
@@ -0,0 +1,127 @@
+# Sui ITS v1.1.3
+
+|  | **Owner** |
+|-----------|------------|
+| **Created By** | @blockchainguyy <ayush@interoplabs.io> |
+| **Deployment** | @blockchainguyy <ayush@interoplabs.io> |
+
+| **Network** | **Deployment Status** | **Date** |
+|-------------|----------------------|----------|
+| **Devnet Amplifier** | - | TBD |
+| **Stagenet** | - | TBD |
+| **Testnet** | - | TBD |
+| **Mainnet** | - | TBD |
+
+[Release](https://github.com/axelarnetwork/axelar-cgp-sui/releases/tag/v1.1.3)
+
+[Changelog](https://github.com/axelarnetwork/axelar-cgp-sui/blob/v1.1.3/CHANGELOG.md)
+
+## Deployment
+
+- Ensure that ITS Hub has been upgraded to [v1.1.0](../cosmwasm/2025-01-ITS-v1.1.0.md) first.
+- The rollout involves deploying Sui ITS contracts version `v1.1.3` on Sui.
+
+```bash
+# Clone latest main and update deps
+npm ci
+```
+
+Create an `.env` config. Use `all` for `CHAINS` to run the cmd for every EVM chain, or set a specific chain.
+
+```yaml
+PRIVATE_KEY=<sui-deployer-key>
+PRIVATE_KEY_TYPE="mnemonic" # Optional
+SIGNATURE_SCHEME=secp256k1
+ENV=<devnet-amplifier|stagenet|testnet|mainnet>
+CHAIN=sui
+```
+
+### Deploy move contracts
+- Sync move directories
+```bash
+node sui/deploy-contract.js sync
+```
+
+- Delete existing gas `GasService` & `Example` config from `$ENV.json`
+
+- Deploy GasService contract
+```bash
+node sui/deploy-contract.js deploy GasService
+```
+
+- Transfer cap to Operators contract
+```bash
+node sui/operators.js storeCap
+```
+
+- Deploy InterchainTokenService and Example contract
+```bash
+node sui/deploy-contract.js deploy InterchainTokenService
```

Comment:
`Abi` needs to be deployed as an immutable package as a separate step. `Utils` is already deployed, so it can be reused as-is for now. Maybe it can be made immutable even if it's upgradable atm, you can take a look

Reply:
`Utils` can be kept as-is, but can `Abi` be deployed with immutable flag set, it's already supported, right?